modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -31,41 +31,70 @@ endif::[]
 
 . Stop all of the machine configuration pools managed by the Machine Config Operator (MCO):
 
-** Stop the master configuration pool:
+** Stop the `master` configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool master --type='merge' --patch \
   '{ "spec": { "paused": true } }'
 ----
 
-** Stop the worker machine configuration pool:
+** Stop the `worker` machine configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool worker --type='merge' --patch \
   '{ "spec":{ "paused": true } }'
 ----
 
 . To prepare for the migration, set the migration field to `null` by entering the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": null } }'
 ----
 
-. To start the migration, set the network plugin back to OpenShift SDN by entering the following commands:
+. Apply the patch to the `Network.operator.openshift.io` object to set the network plugin back to OpenShift SDN by entering the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": { "networkType": "OpenShiftSDN" } } }'
+----
+
+. Confirm that the `Network.operator.openshift.io cluster` object's network plugin is `OpenShiftSDN` by entering the following command. If the command output does not show `OpenShiftSDN`, wait a few seconds for the patch operation to complete.
```

Comment:
We shall also add this step to the OVN->SDN migration doc.

Reply:
This is the rollback doc. We have a reverse migration document at https://docs.openshift.com/container-platform/4.14/networking/openshift_sdn/migrate-to-openshift-sdn.html

Reply:
Ah, yes. I see now. The file is shared amongst both of these docs, so the same updates will apply.

![image](https://github.com/openshift/openshift-docs/assets/57954076/b40bc664-7fb7-4a28-8f6b-be5e794074b6)

Reply:
OK. I figured out what happened here as I could not find the doc in 4.15+. This 4.15 note details the why:

![Screenshot from 2024-06-06 10-58-58](https://github.com/openshift/openshift-docs/assets/57954076/321b79b0-22cd-4d41-b86a-5f80cfbeb0b9)

Reply:
I'll need to update the nw-ovn-kubernetes-migration.adoc file in 4.14 and earlier release through a separate PR. After I get #76876 merged, I'll create the follow-up PR.

Reply:
The same file is referenced in the 4.14 docs so the update will already apply. We just cannot get a preview link on this PR as it targets the `main` branch to which matches 4.16.

![Screenshot from 2024-06-06 11-09-18](https://github.com/openshift/openshift-docs/assets/57954076/6513a872-48b8-4537-9d45-01cf611bc32a)